### PR TITLE
fix(resource_alicloud_db_database):fix api rds resource bug

### PR DESCRIPTION
**Pull request by zhaiyongsheng0123**
fix(resource_alicloud_db_database):fix api rds resource bug;

`=== RUN   TestAccAlicloudDBDatabaseUpdate
--- PASS: TestAccAlicloudDBDatabaseUpdate (641.38s)
PASS`

### CODE REVIEW

**Comment by zhaiyongsheng0123** [comment]
fix api rds resource bug;

**Comment by zhaiyongsheng0123** [comment]
fix api rds resource bug;

File: alicloud/resource_alicloud_db_database.go [hunk 1]
```diff
@@ -29,7 +29,6 @@ func resourceAlicloudDBDatabase() *schema.Resource {
 
 			"name": {
 				Type:     schema.TypeString,
-				ForceNew: true,
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_db_database.go]: Please add update support!

File: alicloud/resource_alicloud_db_database.go [hunk 2]
```diff
@@ -38,7 +37,6 @@ func resourceAlicloudDBDatabase() *schema.Resource {
 				ValidateFunc: validateAllowedStringValue(CHARACTER_SET_NAME),
 				Optional:     true,
 				Default:      "utf8",
-				ForceNew:     true,
```
Comment (ddaddaddda) [inline alicloud/resource_alicloud_db_database.go]: Please add update support!

File: website/docs/r/db_database.html.markdown [hunk 3]
```diff
@@ -62,13 +62,16 @@ resource "alicloud_db_database" "default" {
 The following arguments are supported:
 
 * `instance_id` - (Required, ForceNew) The Id of instance that can run database.
-* `name` - (Required, ForceNew) Name of the database requiring a uniqueness check. It may consist of lower case letters, numbers, and underlines, and must start with a letter
+* `name` - (Required) Name of the database requiring a uniqueness check. It may consist of lower case letters, numbers, and underlines, and must start with a letter
                       and have no more than 64 characters.
-* `character_set` - (Required) Character set. The value range is limited to the following:
+                      
+-> **NOTE:** The current field `name` does not support modification.
+* `character_set` - (Optional) Character set. The value range is limited to the following:
```
Comment (ddaddaddda) [inline website/docs/r/db_database.html.markdown]: Please make sure the name and character_set  support update!

**Pull request closed without merging**